lib/templates/elements/futurism_utils.js

```diff
@@ -26,7 +26,9 @@ const observerCallback = (entries, observer) => {
 
 export const extendElementWithIntersectionObserver = element => {
   Object.assign(element, {
-    observer: new IntersectionObserver(observerCallback.bind(element), {})
+    observer: new IntersectionObserver(observerCallback.bind(element), {
+      rootMargin: '30px 0px 30px 0px'
```

Comment:
why not add `30px` on all sides?

Reply:
I'm not yet attached to 30px but I definitely can't fathom intersecting from the left or right, when do you do that in a normal app?

Reply:
Horizontal sliders! That’s actually where I tested it first.

Reply:
Also I think, sensible defaults are good, but we should make this configurable to avoid unpleasant surprises